src/gpodder/gtkui/model.py

```diff
@@ -494,6 +496,7 @@ def get_update_fields(self, episode):
                 self.C_TOTAL_TIME_AND_SIZE, episode.total_time,
                 self.C_FILESIZE_AND_TIME_TEXT, "%s\n<small>%s</small>" % (filesize if episode.file_size > 0 else "", time),
                 self.C_FILESIZE_AND_TIME, episode.file_size,
+                self.C_TIME_AND_PUBLISHED_TEXT, episode.get_play_info_string() + '\n' + episode.cute_pubdate(),
```

Comment:
What is the purpose of this? Wrapping the fields in a list instead of escaping each line is fine to add here, but I'd rather not have this field lagging the desktop client for a column only available in the adaptive branch. If this is for a column and it makes sense in the desktop then please open another PR for it.

`episode.get_play_info_string()` should use the existing `time` variable to avoid building that string a second time which slows down the update.

Reply:
Yeah, this was a bit questionable. This field is used in adaptive, but in master it's not necessary. I removed this patch.